src/js/utils/i18n.ts

```diff
@@ -11,13 +11,39 @@ export const addTranslation = (
   translationsLanguages[langCode] = languageDictionary;
 };
 
-const getBrowserLanguage = () => globalThis.navigator?.language.split('-')[0] || 'en';
+export class I18n {
```

Comment:
is there an advantage to using a singleton instead of a simple variable in the module?

an ESM module is only imported and executed once so using a simple variable to hold the state is the same I think with less code.

Reply:
It's mostly for my own clarity, but I understand that this is not a pattern used within media-chrome, so I hold my opinions loosely 😄 So I'm happy to convert it to a simple variable.

Reply:
yes, this feels more fitting. thank you. the bundle size is also starting to increase so we have try to squeeze every byte out of it 😄